docs/synapse/devguides/devops_cell.rst

```diff
@@ -0,0 +1,75 @@
+Cell Operations
+===============
+
+As detailed in :ref:`dev_architecture`, the ``Cell`` implements a number of core management functionalities,
+and is therefore used as the base class for Synapse applications.  It is also recommended to implement ``Cell`` for custom services.
+
+.. _devops-cell-config:
+
+Configuring a Cell Service
+--------------------------
+
+Parameters
+**********
+
+A Cell has a set of base configuration parameters that will also be inherited by any implementations:
+
+- ``dirn``: The storage directory for the service which is a required argument for service startup.
```

Comment:
technically, dirn, telepath, https and name are *not* confdefs and we should be careful not to mix them up here.